fdbserver/Ratekeeper.actor.cpp

```diff
@@ -384,7 +385,7 @@ void updateRate(RatekeeperData* self, RatekeeperLimits* limits) {
 	// Look at each storage server's write queue and local rate, compute and store the desired rate ratio
 	for(auto i = self->storageQueueInfo.begin(); i != self->storageQueueInfo.end(); ++i) {
 		auto& ss = i->value;
-		if (!ss.valid) continue;
+		if (!ss.valid || (self->remoteDC.present() && ss.locality.dcId() == self->remoteDC)) continue;
```

Comment:
There's another use of these at line 539 where we check the (currently disabled) version lag. We should probably ignore them there too.

Once that happens, is the only thing ratekeeper is tracking about remote storage servers the space used for `TotalDiskUsageBytes`? I guess if we need that, we don't have the luxury of disabling the tracker requests for remote storage servers.